regression/cbmc/pragma_cprover_enable_all/main.c

```diff
@@ -15,7 +15,7 @@ int main()
 #pragma CPROVER check push
 #pragma CPROVER check disable "bounds"
 #pragma CPROVER check disable "pointer"
-#pragma CPROVER check disable "div-by-zero"
```

Comment:
This regression test is called "...enable_all," so shouldn't the new check be enabled in addition rather than replace an existing one?

Reply:
There isn't any integer division, so that wouldn't have any effect.

Reply:
Ah, but then this test should be extended to have one (the test is "enable_all", so we should be testing all of them).